gen/google/go.mod

```diff
@@ -1,11 +1,9 @@
 module github.com/sourcegraph/controller-cdktf/gen/google
 
-go 1.20
+go 1.18
 
 require (
-	github.com/aws/constructs-go/constructs/v10 v10.1.167
-	github.com/aws/jsii-runtime-go v1.79.0
+	github.com/aws/jsii-runtime-go v1.87.0
 	github.com/hashicorp/terraform-cdk-go/cdktf v0.16.3
+	github.com/aws/constructs-go/constructs/v10 v10.2.69
```

Comment:
runtime dependencies are determined by `jsii` generator for Go, it's out of our contorl